data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg

```diff
@@ -482,19 +482,19 @@
             message= _ "Mission? What mission?"
         [/message]
     [/event]
-    
+
     [event]
         name=time over
         [message]
             speaker= narrator
-            message= _ "Sounds of drums and horns echoes on the side of the mountains... Tallin knew what this sound means: it is too late! A large orc army is coming this way..."
+            message= _ "Sounds of drums and horns echoes on the side of the mountains... Tallin knew what this sound meant: it is too late! A large orc army is coming this way..."
         [/message]
-        
+
         [message]
             speaker=Tallin
             message= _ "If only we were faster escaping into the mines this wouldn't have happened..."
```

Comment:
I think it's better to merge this line with the next one, it doesn't need to be 2 separate messages.

Reply:
thanks @nemaara